frontend/packages/dev-console/src/components/pipelines/pipeline-topology/TaskListNode.tsx

```diff
@@ -23,7 +23,10 @@ const taskToOption = (task: PipelineResourceTask, callback: NewTaskNodeCallback)
   };
 };
 
-const TaskListNode: React.FC<{ element: Node }> = ({ element }) => {
+const TaskListNode: React.FC<{ element: Node; unselectedText?: string }> = ({
```

Comment:
When adding the generic type in frontend/packages/dev-console/src/components/pipelines/pipeline-topology/InvalidTaskListNode.tsx you should add the type here as well.

`element: Node<NodeModel, TaskListNodeModelData>`

In line 38 the `as TaskListNodeModelData` casting is not necessary then.